src/entities/Staff.java

```diff
@@ -444,13 +444,13 @@ private void updateMovieListing(ArrayList<Movie> movieDB) {
 					break;
 
 				case 9:
-					DateTimeFormatter formatter = DateTimeFormatter.ofPattern("EEEE, MMM dd, yyyy HH:mm:ss a");
-					LocalDate showtime = null;
+					DateTimeFormatter formatter = DateTimeFormatter.ofPattern("EEEE, MMM dd, yyyy HH:mma");
+					LocalDateTime showtime = null;
 					exit = false;
 					while(!exit) {
 						try {
-							System.out.println("Enter new movie release date e.g (Saturday, Jul 14, 2018 14:31:06 PM) : ");
-							showtime = LocalDate.parse(sc.nextLine(), formatter);
+							System.out.println("Enter new movie release date e.g (Saturday, Jul 14, 2018 14:30PM) : ");
```

Comment:
The other instance of LocalDate was for the movieRelease date which is irrelevant to the showtime. So this is the only instance in staff file